Pod/Classes/PINRemoteImageManager.m

```diff
@@ -508,6 +508,21 @@ - (NSUUID *)downloadImageWithURL:(NSURL *)url
         }
     }
     
+    if ([url.absoluteString hasPrefix:@"data:"]) {
+        NSData *data = [NSData dataWithContentsOfURL:url];
+        if (data) {
+            id object;
+            if ([url.resourceSpecifier isEqualToString:@"image/gif"]) {
```

Comment:
@appleguy  maybe **not** to cache until next major release, which will use hash of url for cache key?